build.sh

```diff
@@ -65,7 +65,7 @@ function configure {
 }
 
 function mak {
-    pushd build && make VERBOSE=1 -j$NPROC
+    pushd build && make -j$NPROC
```

Comment:
Why this change?

Reply:
@Nyrio this is because the length of the log would be more that 10000 lines in travis and then travis cannot shows the logs and any result should be seen in a raw log. That is why we change that to be able to see the travis result.